### PR TITLE
get block_break_animation

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

File: lib/generate-packets.js [hunk 1]
```diff
@@ -56,4 +63,27 @@ async function triggerGameStateChange (server, bot) {
...
+  await once(botOne, 'spawn')
+  bot.on('messagestr', msg => console.log(`client: ${msg}`))
+  bot.chat(`/op ${botOne.username}`)
+  await bot.awaitMessage('Opped tempBot', 'Made tempBot a server operator')
+
+  await botOne.waitForChunksToLoad()
+  console.log('chunks loaded!!')
+  // dig
+  const p1 = once(bot._client, 'block_break_animation')
+  const block = botOne.findBlock({ matching: block => ['grass', 'dirt', 'sand'].some(o => block.name.includes(o)) })
```
Comment (rom1504) [inline lib/generate-packets.js:77]: Why not always dig the block below the bot ?

Comment (rom1504) [reply lib/generate-packets.js:77]: You could use set block to make sure it's something easy to dig

**Comment by rom1504** [comment]
should this be closed ?

**Comment by u9g** [comment]
> should this be closed ?

I'll remake this with a different approach.

**Pull request closed without merging**